.github/pull_request_template.md

```diff
@@ -1,36 +1,36 @@
-## Description
-
 <!---
-Lead with the intended commit body in this description field. For breaking
-changes, please include "BREAKING CHANGE:" at the beginning of your commit
-body.  At a minimum, this section should include a bracketed reference to the
-Jira ticket, e.g. "[DET-1234]". When squash-and-merging, copy this directly
-into the description field.
--->
-
-
-
-## Test Plan
+## PR TITLE (Commit Body)
+When squash-and-merging, copy this directly into the description field.
 
+Check the "Example Commit Body" for conventional commit semantics.
+For breaking changes, please lead with "BREAKING CHANGE:".
```

Comment:
Previously, this "breaking change" part suggested the PR author to call out the breaking changes in the "description" section (which is now apparently called "commentary").

Now this reads as if you're supposed to put "BREAKING CHANGE: " into the PR title which is not the established process.

Reply:
Good point! I'll move that part to the other section. I can keep "description" as well. It looked to be redundant with commentary.

Reply:
ok, yea, I'd personally keep description and get rid of the optional commentary instead of promoting commentary in place of description because I don't see a point in "swapping" these two.